### PR TITLE
fix: add Eric-Obeng

### LINKED ISSUE #1870: Eric Obeng
### Profile screenshot

Screenshot (55)

### PR DESCRIPTION
**Pull request by Eric-Obeng**
closes #1870 

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

## Other additions/changes

- [ ] Changes made in this PR are mentioned below
- [ ] Search previous suggestions before making a new one, as yours may be a duplicate.
- [ ] I acknowledge that all my contributions will be made under the project's license.
- [ ] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: Eric-Obeng.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://github.com/EddieHubCommunity/awesome-github-profiles/assets/61484602/c870b265-48ff-414d-aabe-45b004f21326",
+    "issueId": "1870",
+    "name": "Eric Obeng",
+    "username": "Eric-Obeng"
+}
```
Comment (Panquesito7) [inline Eric-Obeng.json:6]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

File: Eric-Obeng.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://github.com/EddieHubCommunity/awesome-github-profiles/assets/61484602/c870b265-48ff-414d-aabe-45b004f21326",
+    "issueId": "1870",
```
Comment (Panquesito7) [inline Eric-Obeng.json:3]:
```suggestion
    "issueId": 1870,
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by Eric-Obeng** [comment]
Changes done, please review it.

File: Eric-Obeng.json (hunk 1 above)
Comment (Panquesito7) [reply Eric-Obeng.json:6]: Looks like this is still missing. Do you need any help with this? 🙂

Comment (Eric-Obeng) [reply Eric-Obeng.json:6]: Please what actually is missing

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Pull request merged**